field.go

```diff
@@ -1,6 +0,0 @@
-package main
```

Comment:
i agree it makes the yml files look very clean now but am worried about long term extensibility here; basically if we merge this PR as is and need to add a field to `Field` in the future (which seems decently likely), it won't be a very pretty refactor and will break all existing customer `schema.json`s. thoughts here?